src/module/canvas/layer/template.ts

```diff
@@ -15,11 +15,11 @@ export class TemplateLayerPF2e<
 
     /** Set a grid-snapping mode appropriate for an effect area type */
     snapFor(areaShape: EffectAreaShape | null): void {
+        const M = CONST.GRID_SNAPPING_MODES;
         if (areaShape && canvas.grid.type === CONST.GRID_TYPES.SQUARE) {
-            this.#snappingMode =
-                areaShape === "burst" ? CONST.GRID_SNAPPING_MODES.CORNER : CONST.GRID_SNAPPING_MODES.CENTER;
+            this.#snappingMode = areaShape === "burst" ? M.CORNER : M.CENTER;
         } else {
-            this.#snappingMode = CONST.GRID_SNAPPING_MODES.CENTER;
+            this.#snappingMode = M.CENTER | M.CORNER;
```

Comment:
That doesn't look right.

Reply:
They're bitwise flags,that's how you say "center or corner".

Reply:
If you look at placeables-layers.js at getSnappedPoint() you'll see them being combined in that way and I was like "oh, that's how you're supposed to use it...."

Reply:
Oh, yeah. That's a thing.